src/EditorFeatures/Core/Shared/Options/FeatureOnOffOptions.cs

```diff
@@ -8,71 +8,32 @@ namespace Microsoft.CodeAnalysis.Editor.Shared.Options
-
-        public static readonly PerLanguageOption2<bool> Outlining = new("FeatureOnOffOptions_Outlining", defaultValue: true);
-
-        public static readonly PerLanguageOption2<bool> KeywordHighlighting = new("FeatureOnOffOptions_KeywordHighlighting", defaultValue: true);
-
-        public static readonly PerLanguageOption2<bool> ReferenceHighlighting = new("FeatureOnOffOptions_ReferenceHighlighting", defaultValue: true);
-
-        public static readonly PerLanguageOption2<bool> AutoInsertBlockCommentStartString = new("FeatureOnOffOptions_AutoInsertBlockCommentStartString", defaultValue: true);
-
-        public static readonly PerLanguageOption2<bool> PrettyListing = new("FeatureOnOffOptions_PrettyListing", defaultValue: true);
-
-        public static readonly PerLanguageOption2<bool> StringIdentation = new("FeatureOnOffOptions_StringIdentation", defaultValue: true);
-
-        public static readonly PerLanguageOption2<bool> RenameTrackingPreview = new("FeatureOnOffOptions_RenameTrackingPreview", defaultValue: true);
-
         /// <summary>
         /// This option is not currently used by Roslyn, but we might want to implement it in the
         /// future. Keeping the option while it's unimplemented allows all upgrade paths to
         /// maintain any customized value for this setting, even through versions that have not
         /// implemented this feature yet.
         /// </summary>
-        public static readonly PerLanguageOption2<bool> RenameTracking = new("FeatureOnOffOptions_RenameTracking", defaultValue: true);
+        public static readonly PerLanguageOption2<bool> RenameTracking = new("dotnet_feature_on_off_options_rename_tracking", defaultValue: true);
 
         /// <summary>
         /// This option is not currently used by Roslyn, but we might want to implement it in the
         /// future. Keeping the option while it's unimplemented allows all upgrade paths to
         /// maintain any customized value for this setting, even through versions that have not
         /// implemented this feature yet.
         /// </summary>
-        public static readonly PerLanguageOption2<bool> RefactoringVerification = new("FeatureOnOffOptions_RefactoringVerification", defaultValue: false);
-
-        public static readonly Option2<bool> NavigateAsynchronously = new("FeatureOnOffOptions_NavigateAsynchronously", defaultValue: true);
-
-        /// <summary>
-        /// This option was previously "bool?" to accomodate different supported defaults
-        /// that were being provided via remote settings. The feature has stabalized and moved
-        /// to on by default, so the storage location was changed to
-        /// TextEditor.%LANGUAGE%.Specific.AddImportsOnPaste2 (note the 2 suffix).
-        /// </summary>
-        public static readonly PerLanguageOption2<bool> AddImportsOnPaste = new("FeatureOnOffOptions_AddImportsOnPaste", defaultValue: true);
```

Comment:
This one has the corresponding flight
https://targetednotifications.vsdata.io/rule/77b49a96-879b-454e-bfae-2b572b59eb0a
But the comment says it's not in experiment. tag @ryzngard  to verify